mtp_api/apps/mtp_auth/models.py

```diff
@@ -21,7 +32,16 @@ class ApplicationUserMapping(TimeStampedModel):
     application = models.ForeignKey('oauth2_provider.Application')
 
     def __str__(self):
-        return self.user.username
+        return '%s -> %s' % (self.user.username, self.application.client_id)
+
+
+class ApplicationGroupMapping(TimeStampedModel):
```

Comment:
indeed, don't think could auto load applications as they aren't entirely repeatable (user/secret)

i see your point in that it might be nice if at least for the tests they were loaded from a fixture rather than generated by code...although again there's the user link requirement...i guess we'd need a basic user fixture as well

certainly i'm not opposed to the idea but i'd prefer to leave it out of this PR